### PR TITLE
Feature/basic weather data

### PR DESCRIPTION
**Pull request by RukshanJS**

### CODE REVIEW

File: src/app/@components/privacy-policy/privacy-policy.component.html
Comment (lihini) [inline src/app/@components/privacy-policy/privacy-policy.component.html:1]: Please replace this with a dummy file for now. Will be providing with a new policy later. Same goes for the terms and conditions as well

Comment (RukshanJS) [reply src/app/@components/privacy-policy/privacy-policy.component.html:1]: @lihini done. please check

File: src/app/@theme/styles/styles.scss
Comment (lihini) [inline src/app/@theme/styles/styles.scss:1]: These aren't necessary changes right? The style linter you used within your IDE should be conflicting with the project's linting rules. Let's follow the rules of the project. Please re enable the prepush hook, fix them, and push again.

File: src/app/@components/privacy-policy/privacy-policy.component.html
Comment (lihini) [inline src/app/@components/privacy-policy/privacy-policy.component.html:1]: @RukshanJS Please replace this with a dummy file for now. Will be providing with a new policy later. Same goes for the terms and conditions as well

File: src/app/@theme/layouts/index.ts
Comment (lihini) [inline src/app/@theme/layouts/index.ts:1]: Let's revert these changes too. You are using VS code as your IDE right? You can keep the linting styles you had before as it is and configure a workspace separate for this project. That way you won't have to change all your configurations. In that workspace configure it to get the tslint rules from the project's tslint.json.

It looks like the changed file count would be reduced a lot after doing this.

**Review by lihini — CHANGES_REQUESTED** [review]

File: src/app/@theme/styles/styles.scss
Comment (RukshanJS) [reply src/app/@theme/styles/styles.scss:1]: fixed

File: src/app/@components/privacy-policy/privacy-policy.component.html
Comment (RukshanJS) [reply src/app/@components/privacy-policy/privacy-policy.component.html:1]: done

File: src/app/@theme/layouts/index.ts
Comment (RukshanJS) [reply src/app/@theme/layouts/index.ts:1]: done

**Pull request merged**